From 830ca57775fb2da7eaf85669670735152972de75 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Sun, 4 Aug 2013 15:39:24 +0800
Subject: [PATCH 238/256] ENGR00273862 ipu lib: remove useless pthread.h

This patch removes useless pthread.h and the
'-lpthread' linking option.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 ipu/Makefile               | 2 +-
 ipu/mxc_ipu_hl_lib.c       | 3 +--
 ipu/mxc_ipu_hl_lib_dummy.c | 3 +--
 ipu/mxc_ipu_lib.c          | 3 +--
 4 files changed, 4 insertions(+), 7 deletions(-)

diff --git a/ipu/Makefile b/ipu/Makefile
index 97241e3..2c22d72 100644
--- a/ipu/Makefile
+++ b/ipu/Makefile
@@ -28,7 +28,7 @@ all install:
 endif
 
 $(LIBNAME).so.$(SONAMEVERSION): $(OBJS)
-	$(CC) -shared -nostartfiles -Wl,-soname,$@ -o $@ $^ -lpthread
+	$(CC) -shared -nostartfiles -Wl,-soname,$@ -o $@ $^
 
 $(LIBNAME).so: $(LIBNAME).so.$(SONAMEVERSION)
 	ln -s $< $@
diff --git a/ipu/mxc_ipu_hl_lib.c b/ipu/mxc_ipu_hl_lib.c
index a2d4ef0..4e62886 100644
--- a/ipu/mxc_ipu_hl_lib.c
+++ b/ipu/mxc_ipu_hl_lib.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2009-2012 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2009-2013 Freescale Semiconductor, Inc. All Rights Reserved.
  *
  */
 
@@ -33,7 +33,6 @@ extern "C"{
 #include <malloc.h>
 #include <string.h>
 #include <signal.h>
-#include <pthread.h>
 #include <time.h>
 #include <semaphore.h>
 #include <sys/time.h>
diff --git a/ipu/mxc_ipu_hl_lib_dummy.c b/ipu/mxc_ipu_hl_lib_dummy.c
index d75f4b2..33f6ec0 100644
--- a/ipu/mxc_ipu_hl_lib_dummy.c
+++ b/ipu/mxc_ipu_hl_lib_dummy.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2009-2012 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2009-2013 Freescale Semiconductor, Inc. All Rights Reserved.
  *
  */
 
@@ -29,7 +29,6 @@
 #include <malloc.h>
 #include <string.h>
 #include <signal.h>
-#include <pthread.h>
 #include <time.h>
 #include <semaphore.h>
 #include <sys/time.h>
diff --git a/ipu/mxc_ipu_lib.c b/ipu/mxc_ipu_lib.c
index e1c92b6..609ab1e 100644
--- a/ipu/mxc_ipu_lib.c
+++ b/ipu/mxc_ipu_lib.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2011 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2004-2013 Freescale Semiconductor, Inc. All Rights Reserved.
  *
  */
 
@@ -29,7 +29,6 @@
 #include <math.h>
 #include <string.h>
 #include <malloc.h>
-#include <pthread.h>
 #include <sys/time.h>
 #include <sys/stat.h>
 #include <sys/mman.h>
-- 
1.8.0

